packages/component-static-pages/client/pages/AuthorGuide/JournalPolicies.js

```diff
@@ -228,11 +228,12 @@ const JournalPolicies = props => (
     <H2>Competing Interests</H2>
 
     <Paragraph.Reading>
-      Authors, reviewers, and editors are all required to declare any competing interests that might be perceived to 
-      interfere with the objectivity of the presentation or handling of the work. Any relevant patents, patent applications,
-      and products in development that relate to a submission must be disclosed in full, including patent numbers and titles. 
-      For further information on competing
-      interests, see the{' '}
+      Authors, reviewers, and editors are all required to declare any competing
```

Comment:
Why are these showing up as changes here? Looks like something off with the develop -> branch?

Reply:
Yeah I'm just looking at those - I think it might be because I did a `git merge origin/develop` and not a `git rebase origin/develop` so it added stuff from develop into that branch